teams/rustdoc-frontend.toml

```diff
@@ -17,6 +20,12 @@ orgs = ["rust-lang", "rust-lang-nursery"]
 extra-teams = ["rustdoc"]
 
 [website]
-name = "Rustdoc team contributors"
-description = "Contributing to Rustdoc on a regular basis"
+name = "Rustdoc web frontend"
+description = "Rustdoc frontend design and development"
```

Comment:
Should we mention the terms UI, UX and/or HTML somewhere here? Or just "frontend" to "web frontend" to disambiguate it?

/me having a giggle about the fact that rustdoc's (web) frontend is actually one of rustdoc's backends from a compiler perspective.

Reply:
just wait till rustdoc has a seperate team responsible for cleaning from HIR and metadata :P